pkg/apis/metal3/v1alpha1/baremetalhost_types_test.go

```diff
@@ -37,22 +37,6 @@ func TestHostAvailable(t *testing.T) {
 			Expected:    false,
 			FailMessage: "host with error returned as available",
 		},
-		{
```

Comment:
Why remove this test case?

Reply:
A previous PR that added the consumerRef field added a redundant test in parallel. This one references the field that was removed, so the test can be removed without losing coverage.